app/src/test/java/homer/common/TemperatureTest.java

```diff
@@ -10,6 +10,7 @@
  */
 public final class TemperatureTest {
     private final TemperatureFactory factory = new TemperatureFactoryImpl();
+    private final double delta = 0.01f;
```

Comment:
Is this a constant? If so i think it should be
```suggestion
    private final static double DELTA = 0.01f;
```